1_demo_hello.bal

```diff
@@ -6,10 +6,21 @@
 
 import ballerina/http;
 
+// Services, endpoint, resources are built into the language.
+// This one is HTTP (other options include WebSockets, Protobuf, gRPC, etc)
```

Comment:
Not clear whether this is HTTP service or endpoint or resource. So better to change it to -

This one is HTTP **service**